modules/apps/dynamic-data-mapping/dynamic-data-mapping-api/src/main/java/com/liferay/dynamic/data/mapping/util/DDMDisplayRegistryUtil.java

```diff
@@ -42,11 +46,4 @@ protected void activate(BundleContext bundleContext) {
 					ddmDisplay.getPortletId())));
```

Comment:
@GabrielHenriquedev please redo this part by property filtering. I think all DDMDisplay service has the "javax.portlet.name" property, if some one is missing it, add it.

Most DDMDisplay are components, we prefer not to pull up the service to trigger component activation during collecting.

CC @rafaprax please watch out for this kind of things in future pulls, I am planning to make some global changes to delay component based services activation system wide.